### ISSUE TITLE
Docs wrong for (all?) APIs - A challenge for newbies!!

### ISSUE THREAD

**Issue by ramezrafla** [neutralinojs/neutralinojs.github.io#359]
After a lot of struggling, we got our NeutralinoJS + Vite app running. I honestly don't think most newbies will survive the out of date / erroneous docs.

This does not work -- no error, just hangs

```
const result = await window.Neutralino.filesystem.writeFile(
    '/home/ubuntu/test.txt',
    'my file content'
)
```

This works:
```
import { filesystem } from "@neutralinojs/lib"
const result = await filesystem.writeFile(
    '/home/ubuntu/test.txt',
    'my file content'
)
```

**Comment by benjammin4dayz** [comment]
Fair point on the docs— it's definitely frustrating to deal with potentially misleading info while connecting the missing dots.

For context, when using a package manager to handle Neutralino's client library, `window.Neutralino` is undefined unless the project is explicitly importing the Neutralino module and assigning it as a property to the window object, like this:

```js
import * as Neutralino from '@neutralinojs/lib';
window.Neutralino = Neutralino
```

This is in contrast to a "vanilla" Neutralinojs project with no frontend library, where the client library is included as a script tag inside the HTML and made available on the global window object.

Perhaps the distinction between the two modes needs to be made clearer?

**Comment by dhrm1k** [comment]
i know this. the documentation for this project is something i can't even find a good adjective for. i'm having my fair share of trouble working with the filesystem api, and this is what i found on [stack overflow](https://stackoverflow.com/questions/71677910/missing-permission-to-execute-the-native-method-filesystem-readfile). i still haven't solved the problem i was facing, so i don't know if this was a valid solution, but nevertheless.

**Comment by dhrm1k** [comment]
@benjammin4dayz there seems to be trouble with the filesystem api even on the Neutralinojs project with no frontend library.
